events/wc_offmap_events.txt

```diff
@@ -958,4 +958,251 @@ character_event = {
 			}
 		}
 	}
+}
+character_event = {
+	id = WCOME.500
+	picture = GFX_evt_old_god_healing
+	desc = EVTDESC_WCOME.500
+	border = GFX_event_normal_frame_religion
+	
+	is_triggered_only = yes
+	
+	only_playable = yes
+	
+	option = {
+		name = EVTOPTA_WCOME.500
+		trigger = {
+			has_injury_trigger = yes
+		}
+		heal_wounds_magic_effect = yes
+		hidden_effect = {
+			random_list = {
+				15 = {
+					trigger = { can_get_mutation_healing_trigger = yes }
+					character_event = { id = WCVOD.300 }
+				}
+				85 = {}
+			}
+			}
+	}
```

Comment:
Speaking of balance, I would include `heal_wounds_magic_effect` in other options. The injury traits are safe and removed quickly, so it's not ok if you pay the same price for wound healing and disease healing.